package.json

```
@@ -41,8 +41,8 @@
     "node": ">=10"
   },
   "peerDependencies": {
-    "@types/react": "^16.8.0 || ^17.0.0 || ^18.0.0",
-    "react": "^16.8.0 || ^17.0.0 || ^18.0.0"
+    "@types/react": "*",
+    "react": "^16.8.0 || ^17.0.0 || ^18.0.0 || ^19.0.0 || ^19.0.0-rc"
```

Adding a suggestion below for an easy commit now that React 19 is out.

```suggestion
    "react": "^16.8.0 || ^17.0.0 || ^18.0.0 || ^19.0.0"
```